www/src/pages/en/deployment/docker.md

```diff
@@ -98,9 +90,9 @@ COPY . .
 # ENV NEXT_TELEMETRY_DISABLED 1
 
 RUN \
- if [ -f yarn.lock ]; then yarn build; \
- elif [ -f package-lock.json ]; then npm run build; \
- elif [ -f pnpm-lock.yaml ]; then yarn global add pnpm && pnpm run build; \
+ if [ -f yarn.lock ]; then SKIP_ENV_VALIDATION=1 yarn build; \
```

Comment:
Maybe add a note before the details block about this part, since it's probably of the more custom thing we do, rest is pretty standard?

Reply:
Agreed, a comment would be good here

Reply:
Done !